code/game/machinery/computer/timeclock_vr.dm

```diff
@@ -208,8 +208,8 @@
 /obj/machinery/computer/timeclock/proc/checkCardCooldown()
 	if(!card)
 		return FALSE
-	if((world.time - card.last_job_switch) < 15 MINUTES)
-		to_chat(usr, "You need to wait at least 15 minutes after last duty switch.")
+	if((world.time - card.last_job_switch) < 10 MINUTES)
+		to_chat(usr, "You need to wait at least 10 minutes after last duty switch.")
```

Comment:
```suggestion
	var/time_left = 10 MINUTES - (world.time - card.last_job_switch)
	if((time_left > 0)
		to_chat(usr, "You need to wait another [round((time_left/10)/60, 1)] minute\s before you can switch.")
```

I think this should work to implement TFF's suggestion? Probably? You can commit it with the button and then pull this branch and test it for me maybe. >.>;;

Reply:
I'd prefer to have this in a seperate PR. I'll keep this in mind for a near future implementation, not feeling too into it rn